pydantic_evals/pydantic_evals/demo/time_range/test_cases_schema.json

```diff
@@ -0,0 +1,394 @@
+{
+  "$defs": {
+    "ClsDatasetRow": {
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "title": "Name",
+          "type": "string"
+        },
+        "inputs": {
+          "$ref": "#/$defs/TimeRangeInputs"
+        },
+        "metadata": {
+          "title": "Metadata",
+          "type": "object"
+        },
+        "expected_output": {
+          "anyOf": [
+            {
+              "$ref": "#/$defs/TimeRangeBuilderSuccess"
+            },
+            {
+              "$ref": "#/$defs/TimeRangeBuilderError"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "default": null,
+          "title": "Expected Output"
+        },
+        "assessments": {
+          "items": {
+            "anyOf": [
+              {
+                "$ref": "#/$defs/short_assessment_llm_rubric"
+              },
+              {
+                "$ref": "#/$defs/assessment_llm_rubric"
+              },
+              {
+                "$ref": "#/$defs/short_assessment_is_instance"
+              }
+            ]
+          },
+          "title": "Assessments",
+          "type": "array"
+        }
+      },
+      "required": [
+        "name",
+        "inputs",
+        "metadata"
+      ],
+      "title": "TimeRangeDatasetRow",
+      "type": "object"
+    },
+    "KnownModelName": {
+      "enum": [
+        "anthropic:claude-3-7-sonnet-latest",
```

Comment:
will users commit this file to git? this could make updating pydantic-ai annoying

Reply:
I think that's the idea, but I don't follow your point here.

Reply:
oh I see, your point is just that the JSON schema is going to update. Should we just use `str` instead of `KnownModelName`? I do like that it gives autocomplete suggestions though..

Reply:
yes that's my point, not sure what to do about it

Reply:
considering it more, it's not just this type. any time pydantic-evals updates the default assessments, this file will change. if we want this to be committed, then those definitions that we control need to live in a separate file that isn't committed. the easy solution is that this whole thing should be gitignored.